Solution.java

```diff
@@ -0,0 +1,51 @@
+/*MyHashSet Implementation
+Runtime is O(logn)
+Space complexity is O(1). No additional space is used
+The program took a long time to give the answer(7s) 
+Stats from leetcode - Is this increase because of using recursion?
```

Comment:
Could be yes, but do not worry about it.
As long as the Asymptotic Time Complexity is log(n).